### PR TITLE
Create link meta tags for alternate languages

### PR DESCRIPTION
**Pull request by jeroendesloovere**
Fix for #1866

### CODE REVIEW

File: src/Frontend/Core/Engine/Page.php [hunk 1]
```diff
@@ -490,6 +490,25 @@ protected function processPage()
...
+            $url = Navigation::getURL($this->pageId, $language);
+
+            // Convert relative to absolute url
+            if (substr($url, 0, 1) == '/') {
+                $url = SITE_URL . $url;
+            }
+
+            // Add hreflang
+            $this->header->addLink(
+                array('rel' => 'alternate', 'hreflang' => $language, 'href' => $url)
```
Comment (carakas) [inline src/Frontend/Core/Engine/Page.php]: you shouldn't add it if the link isn't found

**Comment by carakas** [comment]
@jeroendesloovere can you fix the error that popped up in the tests?

**Comment by jeroendesloovere** [comment]
Nope, can't fix it. Seems like a core fork cms bug.

**Comment by carakas** [comment]
replaced by #1885 since it is based on this one but with passing tests

**Pull request closed without merging**